Backend/dev/docker-compose.yml

```diff
@@ -27,6 +27,8 @@ services:
       - ./local-testing-data/rider-dashboard.sql:/docker-entrypoint-initdb.d/2-rider-dashboard-testing-data.sql:Z
       - ./sql-seed/provider-dashboard-seed.sql:/docker-entrypoint-initdb.d/1-provider-dashboard-seed.sql:Z
       - ./local-testing-data/provider-dashboard.sql:/docker-entrypoint-initdb.d/2-provider-dashboard-testing-data.sql:Z
+      - ./sql-seed/special-zone-seed.sql:/docker-entrypoint-initdb.d/1-special-zone-seed.sql:Z
+      - ./local-testing-data/special-zone.sql:/docker-entrypoint-initdb.d/2-special-zone.sql:Z
```

Comment:
Note: this file is no more; instead you want to add these volume mounts to the `arion-configuration.nix` file.